lib/src/zap/rdma/zap_rdma.c

```diff
@@ -198,6 +198,7 @@ static int __enable_cq_events(struct z_rdma_ep *rep)
 	struct epoll_event cq_event;
 	cq_event.data.ptr = rep;
 	cq_event.events = EPOLLIN | EPOLLOUT;
+	/* safe to suppress "Syscall param epoll_ctl(event)" valgrind msg here. */
```

Comment:
Let's not put these kinds of comments in the code please.

Reply:
Hi @baallan, I think it would be preferable to add information to the README.md under a valgrind section. What do you think?